examples/actix_example/src/main.rs

```diff
@@ -0,0 +1,235 @@
+// use actix_web::{get, post, web, App, HttpResponse, HttpServer, Responder};
+
+use actix_files as fs;
+use actix_http::{body::Body, Response};
+use actix_web::dev::ServiceResponse;
+use actix_web::http::StatusCode;
+use actix_web::middleware::errhandlers::{ErrorHandlerResponse, ErrorHandlers};
+use actix_web::{
+    delete, error, get, guard, http, middleware, post, web, App, Error, HttpRequest, HttpResponse,
+    HttpServer, Result,
+};
+use listenfd::ListenFd;
+use sea_orm::entity::*;
+use sea_orm::query::*;
+use sea_orm::EntityTrait;
+use serde::{Deserialize, Serialize};
+use std::env;
+use tera::Tera;
+
+mod post;
+pub use post::Entity as Post;
+mod setup;
+
+const DEFAULT_POSTS_PER_PAGE: usize = 25;
+
+struct AppState {
+    db_url: String,
```

Comment:
Is it possible for us to store the DbConn inside the AppState? Right now we are creating a new connection for each request, which certainly isn't the best practice.

Reply:
Yes I agree, I tried to do this without success. With both Rocket and Actix I ran into this issue. Both frameworks expect to get a database pool which they can call on to get a connection, this might be a good feature to add to SeaORM but I'm not sure how to do it.

Reply:
> 是的，我同意，我尝试这样做但没有成功。对于 Rocket 和 Actix，我遇到了这个问题。两个框架都希望获得一个数据库池，他们可以调用它来获得连接，这可能是添加到 SeaORM 的一个很好的功能，但我不知道如何去做。

I think we should use the `Arc` packaging

Reply:
I think we should use the `Arc` packaging

Reply:
Hello! Can you share a bit more about the approach in your actix-web-async-graphql-sqlx?

Reply:
> I think we should use the `Arc` packaging

Thanks Oodachi for the hint! I've committed a version using Arc to pass the database connection, all the requests will use the same connection in the case.

Reply:
> Hello! Can you share a bit more about the approach in your actix-web-async-graphql-sqlx?

It seems unnecessary, maybe a custom type is a better choice.

```rust
pub type SharedState = Arc<State>;
```

Reply:
Yes with Actix you can use Arc.. But I don't think you need to. Sqlx's connection is designed to be cloned, so you should be make the connection in fn main, and put the connection in the app data.

Also, make sure to create the pool *outside* `HttpServer::new`, otherwise it'll create a new pool for each thread.

Reply:
My bad. I thought I merged the "Add `clone` trait to `DbConn`" into `0.2.2`. Turn out I didn't. It will be included in `0.2.3` I suppose.